server/container/core/pom.xml

```diff
@@ -128,6 +128,10 @@
             <artifactId>JUnitParams</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.inject</groupId>
+            <artifactId>guice</artifactId>
```

Comment:
Euh no I don't want that dependency everywhere!

What we could have is a static `ContentTypeCleaner.initialize()` method here, and define a guice module `JavaxModule` in `server/container/guice/guice-common` that would be then bound by all products.